hm_13/Stack_manual/Stack_manual/Extention/StackGenericExtention.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using Stack_manual;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Stack_manual.Extention
+{
+   public static class StackGenericExtention
+    {
+            public static int CapacityInt<T>(this StackGeneric<T> stackGeneric)
+            {
+                int result;
+                result = stackGeneric.Lenght-1 - stackGeneric.GetIndex;
+                return result;
+            }
+
+        public static void CapacityString<T>(this StackGeneric<T> stackGeneric)
+        {
+            int result;
+            result = stackGeneric.Lenght-1 - stackGeneric.GetIndex;
+            Console.ForegroundColor = ConsoleColor.Magenta;
```

Comment:
you can use this extensions not only in Console project, so, you do not need to use any operations with Console class to avoid strong references